Http/frontendRoutes.php

```diff
@@ -4,8 +4,8 @@
 
 /** @var Router $router */
 
-$router->group(['prefix' => 'blog'], function (Router $router) {
+$router->group(['prefix' => trans('blog::settings.blog-slug')], function (Router $router) {
     $locale = LaravelLocalization::setLocale() ?: App::getLocale();
-    $router->get('posts', ['as' => $locale . '.blog', 'uses' => 'PublicController@index']);
-    $router->get('posts/{slug}', ['as' => $locale . '.blog.slug', 'uses' => 'PublicController@show']);
+    $router->get(trans('blog::settings.posts-slug'), ['as' => $locale . '.blog', 'uses' => 'PublicController@index']);
```

Comment:
I am not sure, but I think it would not be the same for the route name `en.blog` as it would be for `fr.blog`.

A language switcher may need this information to switch between languages without losing the current scope.